### PR TITLE
Update Slide 6 addyapproved

### LINKED ISSUE #6: Add a Caption To an Existing Slide
## Can you think of a better caption?
Each slide can have more than one caption. Check out the current slide show and add a caption (or two).
If you need help, check out the Appendix in the training manual.

### PR DESCRIPTION
**Pull request by addyapproved**
## My Pull Request
This PR updates the caption of slide 6.

This PR closes / fixes / resolves #6 

## To do
- [x] merge the branch
- [x] delete the ## My Pull Request

### CODE REVIEW

File: _slides/02-addyapproved.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
```
Comment (addyapproved) [inline _slides/02-addyapproved.md:4]: This is a change

File: _slides/02-addyapproved.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
-This is a new caption
+New caption
```
[the remark this answers is not in the document]
Comment (addyapproved) [reply _slides/02-addyapproved.md:6]: It looks good! Have a thought doing it a different way?

Comment (addyapproved) [inline _slides/02-addyapproved.md:6]: Add more details to caption, it will help the other users

Comment (Adomonir) [reply _slides/02-addyapproved.md:6]: Yes try it

File: _slides/02-addyapproved.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
 ![cooler-octocat](https://octodex.github.com/images/twenty-percent-cooler-octocat.png)
 {: .center}
```
Comment (addyapproved) [inline _slides/02-addyapproved.md:9]: test

**Pull request merged**